tests/test_tier_lookup.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from vulnerable_people_form.tier_lookup.lacode_tier_loookup import (
+    LacodeTierLookup,
+    PATH_TO_RESTRICTIONS_FILE,
+    DEFAULT_NATIONAL_ALERT_LEVEL
+)
+
+
+def test_lacode_to_tier_single_restriction():
+    lacode = "E08000011"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode)
+    assert tier == 3
+
+
+def test_lacode_to_tier_multiple_restriction():
+    lacode = "E08000001"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode)
+    assert tier == 3
+
+
+def test_lacode_to_tier_multiple_restriction_previous_date():
+    lacode = "E08000018"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+
+    # create a date object between 12th and 24th October e.g. 20th Oct, 2020
+    previous_date_20_oct_2020 = datetime.strptime("2020-10-20 12:00", '%Y-%m-%d %H:%M')
```

Comment:
Can we use `freezegun` here rather than optional arguments to the method?

Reply:
Done